.github/workflows/ci.yml

```diff
@@ -56,21 +56,21 @@ jobs:
     needs: get-branch-name
     uses: ./.github/workflows/run_example_use_cases_reusable.yml
     with:
-      cedar_policy_ref: "refs/heads/main" # use the latest commit on main
-      cedar_examples_ref: "${{ github.head_ref }}" # use the current PR's commit
+      cedar_policy_ref: refs/heads/${{ needs.get-branch-name.outputs.branch_name }}
```

Comment:
If this is too hacky, I could also switch to the hardcoded "refs/heads/release/3.1.x".

Reply:
either way seems fine. I wouldn't object to hardcoding, but this should mean we wouldn't need to upadate the CI on this branch if we make a 3.2.x, right?

Reply:
Yes, if I hardcode this, we'll need to update each release branch manually. But that should be easy to maintain -- just `ctrl-f` for the version you want to change.